README.md

```diff
@@ -1,16 +1,18 @@
 ![libSonata Logo](logo/libSonataLogo.jpg)
-C++ / Python reader for SONATA circuit files:
+C++ / Python reader for SONATA reports files:
```

Comment:
remark : don't have Python readers but we can keep it like as we will move them soon